tests/Makefile

```diff
@@ -0,0 +1,25 @@
+# Makefile based on libgeometry test Makefile
+
+CC = gcc
+CFLAGS = -fPIC -Wall -Wextra -O2 -g -I../include/
+RM = rm -f
+BIN = test-spellcheck
+LDLIBS = -lcriterion
+
+SRCS = main.c test_dictionary.c test_mock_trie.c ../src/dictionary.c ../src/mock_trie.c
+OBJS = $(SRCS:.c=.o)
+
+.PHONY: all
+all: ${BIN}
```

Comment:
In some places you use `${VAR}`, and in others you use `$(VAR)`. I don't care which you pick but please be consistent (they mean the exact same thing).